base/applications/regedit/find.c

```diff
@@ -88,14 +88,22 @@ static BOOL CompareName(LPCWSTR pszName1, LPCWSTR pszName2)
 }
 
 static BOOL
-CompareData(
-    DWORD   dwType,
-    LPCWSTR psz1,
-    LPCWSTR psz2)
+CompareData(DWORD dwType, LPCVOID pv1, size_t cb1)
 {
-    INT i, cch1 = wcslen(psz1), cch2 = wcslen(psz2);
-    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ)
+    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ || dwType == REG_MULTI_SZ)
```

Comment:
Same comment as in the first PR: please don't add Multi-SZ unless you _actually_ support it. That would entail (1) ensuring a _double_ null-terminator, and (2) looping through all strings instead of just the first one.
Finding things in only the first element of multi-sz strings is inconsistent and confusing for the user.

Reply:
We can actually support `REG_MULTI_SZ` in this PR.
This function searches the substring beyond `UNICODE_NULL`.